static/resources.json

```diff
@@ -409,6 +409,18 @@
             "url": "https://www.issuehaven.com/"
           }
         ]
+      },
+      "Kite": {
+        "description": "Kite's powerful editor integrations allow you to work uninterrupted on the same screen. Need to see examples and documentation? All the information from Kite's Python documentation is available right in the editor copilot — no web browser needed.",
```

Comment:
This description will not do, if we decide we want to merge this.

Reply:
I have come up with a new description, please give some feedback on it.

"A plugin tool that runs alongside several popular editors. The plugin provides extensive code completion and will give you detailed documentation on-the-fly based on what parts of your code you have selected, whether that be standard library modules or non-standard library modules."

Reply:
I thought the plugin was just an addition to it? There's plenty of other editor plugins which do this.

Reply:
It is a plugin, but it is also a application that runs alongside the editor to provide the documentation. I should improve the description to make that clearer.